packages/hub/src/app/users/[username]/models/[slug]/ModelPage.tsx

```diff
@@ -89,15 +89,14 @@ export const ModelPage: FC<Props> = ({ username, slug, children }) => {
   const { data: session } = useSession();
 
   return (
-    <WithTopMenu>
-      <div className="flex items-center gap-4 max-w-2xl mx-auto">
+    <WithTopMenu addMarginToMainSection={false}>
+      <div
+        className="flex items-center gap-4 px-8 pt-5 pb-4 border-b border-gray-300"
```

Comment:
`pt-5 pb-4` instead of `py-4` is surprising, but I guess it's intentional?

Reply:
Intentional. I messed with it a bit, liked this more.